nixos/modules/config/xdg/portal.nix

```diff
@@ -0,0 +1,38 @@
+{ config, pkgs ,lib ,... }:
+with lib;
+{
+  options.xdg.portal={
```

Comment:
Some slightly odd formatting in this module, can we consistently have spaces between operators, please?